docs/DeveloperGuide.md

```diff
@@ -580,4 +580,30 @@ The following are a few testcases to try out.
     4. Test case: Remove the `questions` key from a topic <br>
        Expected: Application will not launch properly and error message will be displayed.
     5. Other ways to corrupt the data files: Remove other keys or mess up the JSON format (remove the outer array) <br>
-       Expected: Similar to previous.
\ No newline at end of file
+       Expected: Similar to previous.
+       
+#### 3.5.2. Starting a quiz
+
+1. Giving quiz commands with different input formats
+    1. Test case: `quiz n/2 t/oop s/10`
+       Expected: Starts a quiz successfully ith 2 questions from the topic of [OOP](#34-glossary) and a time limit for 10 seconds for each question.
+    2. Test case: quiz n/2 t/oop
+       Expected: Quiz will not start. Error message will show and the ideal format structure will be displayed to user.
+    3. Test case: quiz n/2 hi t/oop s/5   
+       Expected: Quiz will not start. Error message will show and the ideal format structure will be displayed to user.
+    4. Other ways to start quiz in wrong format: Use the quiz command but give insufficient inputs or invalid inputs
+       Expected: Quiz will not start. Error message will show and the ideal format structure will be displayed to user.
+       
+2. Trying to start a quiz with invalid inputs
+   1. Test case: quiz n/2 t/oop a/10
+      Expected: Quiz will not start. Error message will show and the ideal format structure will be displayed to user.
+   2. Test case: quiz n/2 t/oop s/0
+      Expected: Quiz will not start. Error message will show to prompt user to give a valid time limit.
+   3. Test case: quiz n/2 t/none s/5
+      Expected: Quiz will not start. Error message will tell user that the topic does not exist in E-Duke-8 and where they can find the available topics.
+   4. Test case: quiz n/0 t/oop s/5
+      Expected: Quiz will not start. Error message will tell user to specify more than 0 quiz questions.
+   4. Other ways to start quiz in wrong format: Use the quiz command but give insufficient inputs or invalid inputs
+      Expected: Similar to previous.
```

Comment:
I think you have to use <br> before every "Expected" so that its on the next line

Reply:
Alright! Fixed, thanks!